docs/build/examples/simple-pinning-service/powergate-lotus-go-ipfs-interactions.md

```diff
@@ -0,0 +1,28 @@
+### Powergate and Its Interactions with Lotus and go-ipfs
+
+Both Lotus and IPFS have several implementations for development in different environments, such as Go, JavaScript, Rust, etc. This tutorial focuses on the Go implementation.
+
+- [go-ipfs](https://github.com/ipfs/go-ipfs) is a Go implementation of the [IPFS protocol](https://ipfs.io/).
+- [Lotus](https://github.com/filecoin-project/lotus) is a Go implementation of the Filecoin protocol.
+
+There are several [protocol implementations](../../core-products/protocol-implementations.md) for Lotus, including implementations in other languages such as Rust ([forest](https://github.com/chainsafe/forest) implementation) and C++ ([fuhon](https://github.com/filecoin-project/cpp-filecoin) implementation). The core Lotus node runs the blockchain system, makes storage and retrieval deals, performs data transfers, supports block producer logic, and syncs and validates the chain. Lotus also provides a separate process for storage mining. Filecoin storage miners contribute to the network by producing sector commitments and _Proofs-of-Spacetime_ to prove they have been correctly storing storage client data. You can read more about the interactions between storage miners and storage clients [here](/introduction/what-is-filecoin).
```

Comment:
wherever possible can you link the item in the sentence instead of using the word "here" as a link. Also all internal links need to resolve to the correct markdown file in order to compile correctly.

correct: `[protocol implementations](../../core-products/protocol-implementations.md)`
incorrect: `[here](/introduction/what-is-filecoin)`

Reply:
I think there are no such "incorrect" links in this tutorial.

Reply:
checking again: I was referring to the `/introduction/what-is-filecoin` link, please make this a relative path to the markdown file otherwise the link can not be navigated from GitHub or detected for 404s on the CI.